### PR TITLE
Metrics Alerting

### PR DESCRIPTION
**Pull request by concreted**
Update alerts to prom2.0 format, set sane alert thresholds, pass email auth. 

Alerts thresholds:
- CPU: >90%
- Memory: <400MB free
- Disk: <25% free

@KiaFathi The alert emails are sent via SMTP. Gmail allows creating an 'app password' to send emails as a given Gmail account. See https://support.google.com/accounts/answer/185833?hl=en for steps. What account should we use to send the alert emails, and which address should they go to? engineering@ for both?

### CODE REVIEW

**Comment by KiaFathi** [comment]
@concreted engineering@ for both.

File: Dockerfile [hunk 1]
```diff
@@ -58,9 +58,10 @@ ENV RESIN_APP_NAME STK1A32SCStaging
 ENV DISCOVERY_INTERVAL 30000
 
 # Alert Manager configs
-ENV GMAIL_ACCOUNT aric@getmira.com
-ENV GMAIL_AUTH_TOKEN youGmailpassword
-ENV THRESHOLD_CPU 50
+ENV GMAIL_ACCOUNT fake@getmira.com
+ENV GMAIL_AUTH_TOKEN CHANGE_ME
+ENV NOTIFICATION_EMAIL fake_notification@getmira.com
```
Comment (jschr) [inline Dockerfile]: how come we need fake values here?

File: scripts/update_eb.sh [hunk 2]
```diff
@@ -18,6 +18,11 @@ case $CI_BRANCH in
     AUTH_PASSWORD=$AUTH_PASSWORD_STAGING \
     RESIN_EMAIL=$RESIN_EMAIL \
     RESIN_PASS=$RESIN_PASS \
+    GMAIL_ACCOUNT=$GMAIL_ACCOUNT \
```
Comment (jschr) [inline scripts/update_eb.sh]: Should this be NOTIFICATION_EMAIL?

File: scripts/update_eb.sh [hunk 3]
```diff
@@ -29,6 +34,11 @@ case $CI_BRANCH in
     AUTH_PASSWORD=$AUTH_PASSWORD_PRODUCTION \
     RESIN_EMAIL=$RESIN_EMAIL \
     RESIN_PASS=$RESIN_PASS \
+    GMAIL_ACCOUNT=$GMAIL_ACCOUNT \
```
Comment (jschr) [inline scripts/update_eb.sh]: Should this be NOTIFICATION_EMAIL?

File: Dockerfile (hunk 1 above)
Comment (concreted) [reply Dockerfile]: i wanted to indicate somewhere obvious that these env vars need to be set. unless you look at the deploy scripts it's hard to know which env vars are used by one of our apps.

File: scripts/update_eb.sh (hunk 2 above)
Comment (concreted) [reply scripts/update_eb.sh]: good catch, i need to add `NOTIFICATION_EMAIL`. `GMAIL_ACCOUNT` is the sender, `NOTIFICATION_EMAIL` is the recipient

**Pull request merged**